frontend/src/concepts/__tests__/StartNotebookModal.spec.tsx

```diff
@@ -76,8 +75,11 @@ describe('Start Notebook modal', () => {
     const stepper = screen.getByTestId('notebook-startup-steps');
     expect(stepper).toBeTruthy();
     const steps = screen.getAllByRole('listitem');
-    expect(steps).toHaveLength(15);
-    expect(steps[1]).toHaveTextContent('Failed to scale-up');
+    expect(steps).toHaveLength(14);
+    /* TODO: This finds the POD_PROBLEM: 'There was a problem with the pod'
+      once text and location of error is finalized with UX set the error location and error text
+    */
+    //expect(steps[3]).toHaveTextContent('Failed to scale-up');
```

Comment:
Add the test now -- no biggie on pushing stuff -- sometimes UX doesn't get to it right away, we can do best effort.